__tests__/app.test.js

```diff
@@ -97,11 +97,77 @@ describe("2.getReviews", () => {
                 }          
             })
         })
-        test("returns a 404 status code if path not valid", () => {
+    })
+})
+
+describe("3.getReviewById", () => {
+    describe("GET /api/reviews/:id", () => {
+        test("returns a single review", () => {
+            return request(app)
+            .get("/api/reviews/3")
+            .expect(200)
+            .then(res => {
+                let review = res.body.review;
+                expect(typeof(review)).toBe("object");
+                expect(review).toMatchObject({
+                    review_id: 3,
+                    title: 'Ultimate Werewolf',
+                    designer: 'Akihisa Okui',
+                    owner: 'bainesface',
+                    review_img_url:
+                      'https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?w=700&h=700',
+                    review_body: "We couldn't find the werewolf!",
+                    category: 'social deduction',
+                    created_at: "2021-01-18T09:01:41.251Z",
+                    votes: 5
+                  })
+            })
+        })
+        test("returns a 404 error if input number does not exist", () => {
+            return request(app)
+            .get("/api/reviews/150")
+            .expect(404)
+        })
+        test("returns a 400 error if input is not number type", () => {
+            return request(app)
+            .get("/api/reviews/abc")
+            .expect(400)
+        })
+    }) 
+})
+
+describe.only("4.getCommentsById", () => {
+    describe("GET /api/reviews/:id/comments", () => {
+        test("returns an array of comments", () => {
+            return request(app)
+            .get("/api/reviews/2/comments")
+            .expect(200)
+            .then(res => {
+                let comments = res.body.comments;
+                expect(Array.isArray(comments)).toBe(true);
+                expect(comments).toHaveLength(3);
+            })
+        })
+        test("returns an empty array if id is valid but no comments are found", () => {
+            return request(app)
+            .get("/api/reviews/1/comments")
+            .expect(200)
+            .then(res => {
+                comments = res.body.comments;
+                expect(comments).toHaveLength(0);
+            })
+        })
+        test("returns a 404 error if input is not found", () => {
             return request(app)
-            .get("/api/review")
+            .get("/api/reviews/47/comments")
             .expect(404)
```

Comment:
You should also test the message that comes back, and if there isn't one you should reject with a message aswell as a status code.